docs/reference/modules/cluster/misc.asciidoc

```diff
@@ -17,6 +17,28 @@ user with access to the <<cluster-update-settings,cluster-update-settings>>
 API can make the cluster read-write again.
 
 
+[[cluster-max-tombstones]]
+==== Index Tombstones
+
+The cluster state maintains index tombstones to explicitely denote indices that 
+have been deleted.  The tombstones are maintained to prevent indices that were 
+deleted when a node was offline from being reimported as dangling indices when 
+the node comes back online.  The number of tombstones maintained in the cluster 
+state is controlled by the following property:
+
+`cluster.indices.tombstones.size`::
+
+The maximum number of tombstones that can be in the cluster state before tombstones 
+are purged.  The current default is 500.  Index tombstones remain in the cluster state 
+forever, until the maximum is exceeded, at which point tombstones are purged to stay 
+within the maximum allowed.  If your use case involves a large amount of regular index 
```

Comment:
> Wikimedia Foundation, for example, when doing updates will have tons of index deletions as part of their process

When we made mapping or analysis changes we'd have to rebuild all the indexes with the new mapping. It was an as needed, somewhat rare thing. We'd reindex, swap the alias, and remove about 1,600 indexes over the course of a week. Some took hours and some seconds.